File: Backend/vietqtran.Api/vietqtran.Models/ResponseModels/CommentResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using vietqtran.Models.Entities.Relations;
using vietqtran.Models.Entities;
using vietqtran.Models.ViewModels;

namespace vietqtran.Models.ResponseModels
{
	public class CommentResponse
	{
		public Guid Id { get; set; }
		public string Content { get; set; }
		public Guid UserId { get; set; }
		public Guid PostId { get; set; }
		public DateTime? ModifiedAt { get; set; }
		public bool? IsReply { get; set; }
		public Guid? ReplyId { get; set; }
		public AppUserVM User { get; set; }
		public ICollection<LikeComment> LikeComments { get; set; }
	}
}

File: ClientApp/type/responseModel/commentResponse.ts
import { User } from "@type/models/User";

export interface CommentResponse {
    id: string;
    content: string;
    userId: string;
    postId: string;
    modifiedAt: string | null;
    isReply: boolean | null;
    replyId: string | null;
    user: User;
    likeComments: string[]
}